public/app/features/explore/Explore.tsx

```diff
@@ -285,26 +288,99 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
     );
   };
 
+  renderGraphPanel = (width: number) => {
+    const { graphResult, loading, absoluteRange, timeZone } = this.props;
+    return (
+      <ExploreGraphPanel
+        ariaLabel={selectors.pages.Explore.General.graph}
+        series={graphResult}
+        width={width}
+        loading={loading}
+        absoluteRange={absoluteRange}
+        isStacked={false}
+        showPanel={true}
+        timeZone={timeZone}
+        onUpdateTimeRange={this.onUpdateTimeRange}
+        showBars={false}
+        showLines={true}
+      />
+    );
+  };
+
+  renderTablePanel = (width: number) => {
```

Comment:
please use prototype methods for all these renderXX functions. Arrow (instance) methods should only be used if they are required for a callback. Both more readable and requires less memory